src/pages/settings/settings.html

```diff
@@ -246,7 +246,7 @@ <h4>Using Tidal Text</h4>
               <div class="group__option" class="hidden">
                 <div class="group__description">
                   <h4>Show song</h4>
-                  <p>Show the current song in the Discord client</p>
+                  <p>Show the current media in the Discord client</p>
```

Comment:
I feel like `Show song` got left out during the renames of `song` to `media` as otherwise the description and title for the feature do not match